javaguide.html

```diff
@@ -1189,7 +1189,6 @@ <h3 id="s7.2-summary-fragment">7.2 The summary fragment</h3>
 incorrect, and should be changed to
 <code class="prettyprint lang-java">/** Returns the customer ID. */</code>.</p>
 
-<a name="s7.3.3-javadoc-optional"></a> 
```

Comment:
Note this is not a link but an anchor, and shouldn't be removed because existing links may still use it.